extensions/nos-news/package.json

```
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "nos-nieuws",
+  "title": "NOS Nieuws",
+  "description": "Used to display the latest NOS nieuws from nos.nl",
+  "icon": "command-icon.png",
+  "author": "jorandob",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Show NOS News",
```

Personally I don't think that "Show" is needed here, but that might be a matter of taste.

```suggestion
      "title": "NOS News",
```

---

I was thinking the show might be nice as I want to add more variations/sources later on. Something like 'Show Lastest News from NOS' or something similar. 😄